openzwave-sys/build.rs

```diff
@@ -1,4 +1,29 @@
 extern crate gcc;
+use std::fs;
+use std::path::Path;
+use std::process::Command;
+use std::env;
+
+fn find_include_path(c: &mut gcc::Config) -> Result<(), ()> {
+    // TODO try to actually build with these paths
+    let possible_paths = vec![
+        "/usr/include/openzwave",
+        "/use/local/include/openzwave",
```

Comment:
If we're building from a sub-module, we shouldn't require the headers to be installed as a dependency, We should be using the headers from the sub-module.

r- for this